### DISCUSSION TITLE
RHEL 9.4 upgrade to 314?

### DISCUSSION THREAD

**Discussion by Cyraxianknight** [cockpit-project/cockpit#20586] [Q&A] [answered]
Hello, I am attempting to upgrade from 311.2 to 314 to address CVE-2024-2947 but so far have been unsuccessful. whats the best way to upgrade to 314? i am on RHEL 9.4 and dnf-upgrade only got me to 311.2. is there a specific command i should run?

**Comment by martinpitt** [comment, marked as answer]
You already have the fix, 311.2 includes it. See https://access.redhat.com/errata/RHSA-2024:3843
